.build/ory/kratos/email-password/kratos.yml

```diff
@@ -4,50 +4,63 @@ dsn: memory
 
 serve:
   public:
-    base_url: http://127.0.0.1:4433/
+    base_url: http://localhost:4433/
     cors:
       enabled: true
   admin:
     base_url: http://kratos:4434/
 
 selfservice:
-  default_browser_return_url: http://127.0.0.1:4455/
+  default_browser_return_url: http://localhost:3000/
   whitelisted_return_urls:
-    - http://127.0.0.1:4455
+    - http://localhost:3000
 
   methods:
     password:
       enabled: true
+    oidc:
+      enabled: true
+      config:
+        providers:
+          - id: microsoft
+            provider: microsoft
+            client_id: feb6d8b1-8cc2-4e7d-a419-ad7b544b0832
```

Comment:
presume this is from our playground env?

Reply:
yes, of course, all is playground config